_posts/2019-05-29-canary-releases.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: Canary Releases with Traefik on GKE at HolidayCheck
+date: 2019-05-29 12:0:00 +0200
+author_name: Periklis Tsirakidis
+author_url : /author/periklistsirakidis
+author_avatar: periklistsirakidis
+read_time : 10
+feature_image: posts/2019-05-29-canary-releases/canary-in-a-coal-mine.jpg
```

Comment:
I would suggest adding `excerpt` here. Otherwise blog's engine will just cut 155 (or so) characters from the beginning of your post.